--- ../src/Providers/Mistral/Handlers/Audio.php
+++ ../src/Providers/Mistral/Handlers/Audio.php
@@ -39,6 +39,9 @@
                 'response_format' => $request->providerOptions('response_format') ?? null,
                 'temperature' => $request->providerOptions('temperature') ?? null,
                 'timestamp_granularities' => $request->providerOptions('timestamp_granularities') ?? null,
+
+                // Patch https://github.com/prism-php/prism/pull/995
+                'diarize' => $request->providerOptions('diarize') ?? null,
             ]));
 
         if (json_validate($response->body())) {
